model/generate.go

```diff
@@ -173,7 +173,7 @@ func cacheProjectData(p *Project) projectMaps {
 }
 
 // saveNewBuildsAndTasks saves new builds and tasks to the db.
-func (g *GeneratedProject) saveNewBuildsAndTasks(cachedProject *projectMaps, v *version.Version, p *Project) error {
+func (g *GeneratedProject) saveNewBuildsAndTasks(cachedProject *projectMaps, v *version.Version, p *Project, parentPriority int64) error {
```

Comment:
Why does int suffice? (The existing priorities are int64s so if an int is int32 on a particular architecture a large priority could get truncated. No?)

Reply:
Didn't realize that we have them as int64s in the model, this can stay as is then. We have validation to prevent values of this field so that int32 is sufficient